dashboard/app/models/levels/free_response.rb

```diff
@@ -53,4 +53,23 @@ def peer_reviewable?
   def icon
     'fa fa-list-ul'
   end
+
+  # FND-985 Create shared API to get localized level properties.
+  # get_property takes a given level property_name and returns the localized
+  # version of it.
+  def get_property(property_name)
+    # Return the default English string from the database model if we shouldn't
+    # localize this property or we couldn't find a localized value.
+    default = try(property_name)
```

Comment:
Ah we moved the source of truth for some levels to the DB because we didn't want to upload everything to Crowdin iirc.

Hopefully as part of a shared API we can define where the English source strings for all levels...